### DISCUSSION TITLE
Installation stuck on Updating Container OS

### DISCUSSION THREAD

**Discussion by Tequila-dev** [tteck/Proxmox#2279] [Help Needed] [answered]
Script installation process stuck on Updating Container OS and never changes. Proxmox VE 8.1.3, default settings, internet connection established. Is there a way to skip the update process and run it later, or maybe any other solution? Thanks in advance.

Logs from the shell:

Using Default Settings
Using Distribution: ubuntu
Using ubuntu Version: 22.04
Using Container Type: 0
Using Root Password: Automatic Login
Using Container ID: 105
Using Hostname: jellyfin
Using Disk Size: 8GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a Jellyfin LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 105 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 10.0.0.140
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.4
 - Updating Container OS...

**Comment by tteck** [comment]
Not stuck, just slow. If there's an issue, it will show an error.

**Reply by Tequila-dev** [reply]
I left the installation process for an entire night, no errors after 8 hours.

**Reply by agenttank** [reply]
Hi!

I am stuck here too with jellyfin script... apt is finished according to /var/log/apt/history.log and no apt process is there anymore. apt isn't locked anymore too.

> 
Using Advanced Settings
Using Distribution: ubuntu
Using ubuntu Version: 22.04
Using Container Type: 1
Using Root Password: 
Container ID: 103
Using Hostname: lxc3
Using Disk Size: 8
Allocated Cores: 4
Allocated RAM: 4096
Using Bridge: vmbr0
Using IP Address: 192.168.3.103/24
Using Gateway IP Address: 192.168.3.254
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using Vlan: 30
Enable Root SSH Access: yes
Enable Verbose Mode: no
Creating a Jellyfin LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local_zfs for Container Storage.
 ✓ Updated LXC Template List
 ✓ Downloaded LXC Template
 ✓ LXC Container 103 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.3.103 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.121.3
 - Updating Container OS...

**Reply by tteck** [reply, marked as answer]
_Hopefully_, this (https://github.com/tteck/Proxmox/commit/00f73c7370716629d111bc8a7148de6224dd5dcb) will cure those symptoms.

**Comment by agenttank** [comment]
it went over it now :) thank you!

**Comment by Tequila-dev** [comment]
Did the trick for me too, but strangely enough only with 23.04 Lunar. On 22.04 or default settings still the same issue.
